airgun/views/hostgroup.py

```diff
@@ -0,0 +1,42 @@
+from widgetastic.widget import Text
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import (
+    ActionsDropdown,
+    FilteredDropdown,
+    SatTable,
+)
+
+
+class HostgroupView(BaseLoggedInView, SearchableViewMixin):
```

Comment:
in general - we do not work like this. If you start your coding for entity, so please implement it completely, not like few buttons you need

Reply:
Don't forget Airgun is here to make Robottelo development easier. And contrary to what you say, there are many missing features in Airgun, even in already implemented entities. Developing UI tests in Robottelo means constantly finding a way to do something while API doesn't support many things and so does not Airgun.

My workflow is that I need to create a certain test and need tools for that - if they do not exist, I have to fix the existing ones. My workflow, contrary to Airgun developer's, is not creating a framework and then writing test based on what is implemented in the framework. I need to write very specific tests and as a test writer, I only do the changes that are necessary in framework.

As an Airgun developer, I'm planning to implement the rest of hostgroup entity later when I have time for that. Right now, however, it is out of scope of this PR. If you need the entity fully implemented immediately, feel free to contribute.

Reply:
Hello @oshtaier . IMO if you need 2 users, you first have to have 1 user. Yes, this is no full HG support in AirGun, yes issue describing what needs to be added to have it complete would be nice (but not required, it can be created by anybody). I do not see a reason why partial HG support could not be committed?

Reply:
@oshtaier , as I said, this doesn't make sense from the test writer point of view. We need to implement tests that test real scenarios. Not to implement tests based on what Airgun can do. Not to implement considerable parts of the framework so we can use some minor functionality. That's not what we are here for.
I've fixed some of the real issues you have pointed out in the PR. Please, review and merge this if there are no further issues.

Reply:
We can't force people to fully implement entities and views if they need only small part of it, as it's out of scope and other parts of entity may appear complex or require some extra knowledge on how to work with them, so i agree with @lhellebr and @jhutar here. But only in case that partial implementation doesn't bring any redundant/temporary code which will add extra work to update/remove it in future.
The only thing we should probably think of is how to mark such entities as "WIP" ones, so others won't get confused why some parts of functionality doesn't work for them.